src/main/java/vg/civcraft/mc/civmodcore/command/CommandHandler.java

```diff
@@ -76,5 +99,21 @@ protected void helpPlayer(Command command, CommandSender sender) {
 		sender.sendMessage(new StringBuilder().append(ChatColor.RED + "Usage: ")
 				.append(command.getUsage()).toString());
 	}
+	
+	public void msg(String msg) {
```

Comment:
What's the purpose of this? All the actual work is already being done by the command itself, I cant think of any use cases where it would make sense to send messages directly over the commandhandle.